pkg/podeffect/pod_effect_test.go

```diff
@@ -0,0 +1,163 @@
+package podeffect
+
+import (
+	"context"
+	"k8s.io/apimachinery/pkg/types"
+	"reflect"
+	"strconv"
+	"testing"
+	"time"
+
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/informers"
+	clientsetfake "k8s.io/client-go/kubernetes/fake"
+	"k8s.io/kubernetes/pkg/scheduler/framework/plugins/defaultbinder"
+	"k8s.io/kubernetes/pkg/scheduler/framework/plugins/queuesort"
+	frameworkruntime "k8s.io/kubernetes/pkg/scheduler/framework/runtime"
+	framework "k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
+	fakeframework "k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1/fake"
+	st "k8s.io/kubernetes/pkg/scheduler/testing"
+	testutil "sigs.k8s.io/scheduler-plugins/test/util"
+)
+
+func TestPodStatusEffect(t *testing.T) {
+
+	tests := []struct {
```

Comment:
Can you expand the nodes number to 3 or 4, so that the score doesn't fall into a binary result: MinNodeScore and MaxNodeScore?